src/main/webapp/app/modules/conflicts/single/details.tsx

```diff
@@ -5,13 +5,15 @@ import { RouteComponentProps, Link } from 'react-router-dom';
 import { IActivityRecord } from 'app/shared/model/activity-record.model';
 import { IOrganizationMatch } from 'app/shared/model//organization-match.model';
 import { OrganizationDetails } from '../shared/components/organization-details';
-import { LocationsDetails } from '../shared/components/location/locations-details';
-import { ServicesDetails } from '../shared/components/service/services-details';
```

Comment:
what is the point of removing these braces?

Reply:
> what is the point of removing these brackets?

Default import is not connected to redux.